internal/gpustats/nvidia.go

```diff
@@ -417,21 +419,42 @@ func GetNvidiaGPUStatus() (NvidiaSmiLog, error) {
 	return ParseNvidiaSmi(output)
 }
 
-// Dummy struct to act as our adapter
-type NvidiaGPUHandler struct{}
+// Struct to act as our adapter
+type NvidiaGPUHandler struct {
+	lookup procinfo.UidLookup
+}
+
+func populateNames(samples *[]uplink.GPUStatSample, lookup procinfo.UidLookup) {
```

Comment:
NIT: You can just take `samples []uplink.GPUStatsSample`, instead of needing a pointer here, as you never modify the slice itself (only it's contents)

Reply:
oh what? good to know that